src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -118,39 +154,40 @@ export const DefinitionList = (props) => {
    * @param {array} data
```

Comment:
Update the docstring with the updated name of the parameter.

Reply:
that's like that, it receives props, the parameters shown are just the serialization of the props